airbyte-integrations/connectors/source-coin-api/integration_tests/configured_catalog.json

```diff
@@ -0,0 +1,13 @@
+{
+  "streams": [
+    {
+      "stream": {
+        "name": "ohlcv_historical_data",
```

Comment:
can you add the second stream too and run the tests again?

Reply:
Yep, oversight on my part, sorry.

![image](https://user-images.githubusercontent.com/47423046/197720632-6bf6f7ec-dca1-4942-958e-94cd2827b017.png)